src/DotNet.Status.Web/Models/IssueMentionForwarder.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using DotNet.Status.Web.TeamsMessages;
+using JetBrains.Annotations;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace DotNet.Status.Web.Models
+{
+    public class IssueMentionForwardingOptions
+    {
+        public string[] WatchedTeams { get; set; }
+        public string[] IgnoreRepos { get; set; }
+        public string TeamsWebHookUri { get; set; }
+    }
+
+    public interface IIssueMentionForwarder
+    {
+        Task<bool> HandleIssueBody([CanBeNull] string oldBody, string newBody, string title, string commentUri, string username, DateTimeOffset date);
```

Comment:
This is a somewhat misleading name. Since it's handling way more than "issuebody".

It's really a "MentionHandler".  The "issue" part isn't really interesting. (It could, technically, scan ANYTHING for a mention).

Reply:
I will change it.